duckdns/data/run.sh

```diff
@@ -65,35 +65,70 @@ fi
 # Run duckdns
 while true; do
 
-    [[ ${IPV4} != *:/* ]] && ipv4=${IPV4} || ipv4=$(curl -s -m 10 "${IPV4}")
-    [[ ${IPV6} != *:/* ]] && ipv6=${IPV6} || ipv6=$(curl -s -m 10 "${IPV6}")
-
-    # Get IPv6-address from host interface
-    if [[ -n "$IPV6" && ${ipv6} != *:* ]]; then
-        ipv6=
-        bashio::cache.flush_all
-        for addr in $(bashio::network.ipv6_address "$IPV6"); do
-	    # Skip non-global addresses
-	    if [[ ${addr} != fe80:* && ${addr} != fc* && ${addr} != fd* ]]; then
-              ipv6=${addr%/*}
-              break
+    ipv4="none"
+    if [[ ${IPV4} != "none" ]]; then
```

Comment:
The default set at the beginning of this script is `""` not `none`.

I think I'd prefer to introduce some additional variables, e.g.

ipv4_enabled=$(bashio::config.has_value "ipv4")

And then use `$ipv4_enabled` here.